pkg/machinery/constants/constants.go

```diff
@@ -84,6 +84,8 @@ const (
 	// KernelParamNetIfnames is the kernel parameter name to control predictable network interface names.
 	KernelParamNetIfnames = "net.ifnames"
 
+	KernelParamISOBootHaltIfInstalled = "talos.iso.boot.halt_if_installed"
```

Comment:
```suggestion
	KernelParamISOBootHaltIfInstalled = "talos.halt_if_installed"
```

Probably shorter to support booting e.g. from PXE?

Reply:
and we need to document this in `kernel.md`